Python/ceval.c

```diff
@@ -3062,6 +3065,37 @@ PyUnstable_Eval_RequestCodeExtraIndex(freefunc free)
     return new_index;
 }
 
+
+static void
+dtrace_function_entry(_PyInterpreterFrame *frame)
+{
+    const char *filename;
+    const char *funcname;
+    int lineno;
+
+    PyCodeObject *code = _PyFrame_GetCode(frame);
+    filename = PyUnicode_AsUTF8(code->co_filename);
+    funcname = PyUnicode_AsUTF8(code->co_name);
+    lineno = PyUnstable_InterpreterFrame_GetLine(frame);
+
+    PyDTrace_FUNCTION_ENTRY(filename, funcname, lineno);
```

Comment:
```suggestion
    PyCodeObject *code = _PyFrame_GetCode(frame);
    const char *filename = PyUnicode_AsUTF8(code->co_filename);
    const char *funcname = PyUnicode_AsUTF8(code->co_name);
    int lineno = PyUnstable_InterpreterFrame_GetLine(frame);
    PyDTrace_FUNCTION_ENTRY(filename, funcname, lineno);
```

Reply:
I'm not sure this nits is necessary, I just recovery some we already have before

FYI https://github.com/python/cpython/blob/dce2d38cb04b541bad477ccc1040a68fa70a9a69/Python/ceval.c#L2976-L3004

Reply:
Oh so it was a simple revert. Ok! don't mind me then.